### PR TITLE
PP-181: Update wallet to use the references to new js package. 

### PR DESCRIPTION
**Pull request by kenny08gt**
# What
Update wallet to use the references to the new js package. Replace zksync js with rif-aggregation release.
This PR depends on https://github.com/rsksmart/ri-aggregation/pull/28 to be merged
# Why
Update the zksync js release with the new changes
# References
https://rsklabs.atlassian.net/browse/PP-181
https://rsklabs.atlassian.net/browse/PP-179

### CODE REVIEW

**Review by jurajpiar — APPROVED** [review]
LG;DT

File: package.json [hunk 1]
```diff
@@ -72,14 +72,14 @@
     "nuxt-build-optimisations": "latest",
     "nuxt-typed-vuex": "latest",
     "oh-vue-icons": "^0.4.7",
+    "rif-aggregation": "^0.11.8",
```
Comment (antomor) [inline package.json:75]: I know this is annoying, but if we change the name as [suggested](https://github.com/rsksmart/ri-aggregation/pull/28), we need to update it here.

**Comment by antomor** [comment]
It depends on the [SDK PR](https://github.com/rsksmart/ri-aggregation/pull/28)

File: package.json (hunk 1 above)
Comment (kenny08gt) [reply package.json]: Updated

**Review by jurajpiar — APPROVED** [review]

File: yarn.lock [hunk 2]
```diff
@@ -3486,6 +3486,7 @@ __metadata:
     "@nuxtjs/sentry": latest
     "@nuxtjs/style-resources": latest
     "@nuxtjs/toast": latest
+    "@rsksmart/rif-aggregation-sdk-js": "https://github.com/rsksmart/ri-aggregation"
```
Comment (antomor) [inline yarn.lock:3489]: I think that this yarn.lock doesn't reflect the latest changes, so we should update it also

**Review by antomor — CHANGES_REQUESTED** [review]
@kenny08gt Thank you for these changes. I think we should just update the yarn.lock and I'm not sure if we need the package-lock.json as well. If not, please let's remove it.

File: yarn.lock (hunk 2 above)
Comment (kenny08gt) [reply yarn.lock]: Updated

**Review by antomor — APPROVED** [review]
LG

**Pull request merged**